### DISCUSSION TITLE
wikijs LDAP issues

### DISCUSSION THREAD

**Discussion by blade1989** [requarks/wiki#2814] [Help / Questions]
We have trouble getting our wikijs to connect with ldap. The output of wikijs on the Linux server, does not show any error messages. 

Here is our LDAP configuration in short:

**LDAP URL**
 `ldap://LDAPHOSTNAME:389`

**Admin Bind DN**
`CN=svc_wiki,OU=svc,OU=OrgName,DC=local,DC=local`
​​
**Search Base**
`OU=SBSUsers,OU=Users,OU=MyBusiness,DC=OrgName,DC=local`
​
**Search Filter**
`(userPrincipalName={{username}})`

**Unique ID Field Mapping**
`userPrincipalName`
​
**Email Field Mapping**
`mail`

Also, is there a way to save LDAP error messages somewhere?

**Comment by pyvarex** [comment]
Seeing the same issues here! Also with the usage of DN in the bind, and usage of uid as username.

No logs could be found, even with the LDAP flags active. Webinterface just says username and password is incorrect, while it isn't.

Help would be much appriciated!

**Reply by mirrorspock** [reply]
I have the same issue, haven't been able to get it to work, th Bind is correct since I only get "Invalid email / username or password" when it is set. but nothing gets logged that I can find.
